letsdrill.rb

```diff
@@ -1,15 +1,23 @@
 def get_letter_grade(integer)
- 
   #Put your code here!
-
+	if integer >= 90
+		return  "A"
+	elsif integer >=80
+		return "B"
+	elsif integer >=70
+		return "C"
+	elsif integer >=60
+		return "D"
+	else
+		return "F"
+	end
 end
 
 def shortest_string(array)
 
   #Put your code here!
-
+  return array.sort_by{|n|n.length}[0]
```

Comment:
Check out [Enumerable#min_by](https://ruby-doc.org/core-2.2.2/Enumerable.html#method-i-min_by). It will get the answer more directly.

Scan through the rest of those Enumerable methods while you're there. Ruby simplifies many common tasks in its core data types.